packages/simcore-sdk/src/simcore_sdk/nodeports/_data_items_list.py

```diff
@@ -0,0 +1,51 @@
+"""This module defines a container for port descriptions"""
+
+# pylint: disable=too-many-ancestors
+import logging
+from typing import Dict
+from collections import MutableMapping
+from simcore_sdk.nodeports import exceptions
+from simcore_sdk.nodeports._data_item import DataItem
+
+log = logging.getLogger(__name__)
+
+class DataItemsList(MutableMapping):
+    """This class contains a list of Data Items."""
+
+    def __init__(self, data:Dict[str, DataItem]=None):

```

Comment:
Very nice, specially all the typing. Try using [attrs](https://www.attrs.org/en/stable/types.html)  with auto_attribs=True next time